### DISCUSSION TITLE
I have a dream i want to be rich

### DISCUSSION THREAD

**Discussion by leslie158** [Benefitspace/dream#3] [Show and tell]
I have a dream i want to be rich
0x06cbda490e96c746234e225f3ed9b06c0e3e14f9

**Comment by westlife29BTC** [comment]
I put 0.1 BNB in it so what next？

**Reply by Benefitspace** [reply]
没事了，问题解决了，我在做优化

**Reply by westlife29BTC** [reply]
那就太棒了。
我的社区很多朋友说上当了，希望能够不要让大家失望。

**Reply by airzhangforce** [reply]
take me rich too ，haha

**Reply by sunShineLoveMe** [reply]
come on! and make it to the moon!!!

**Reply by leslie158** [reply]
> 没事了，问题解决了，我在做优化
0x06cbda490e96c746234e225f3ed9b06c0e3e14f9 终于等到你  希望有空投。为了梦想一起暴富

**Reply by xuzhongnang** [reply]
加油搞起来，一起实现梦

**Reply by hongmeiwu77** [reply]
I want to be rich with rich coin , make a magic dream , 有梦有期待，

**Reply by westlife29BTC** [reply]
有没有推特电报啥的，搞个社区，招募志愿者，网站搞起，各种推广开始，然后就爆发。

**Reply by sean-go-ai** [reply]
> 有没有推特电报啥的，搞个社区，招募志愿者，网站搞起，各种推广开始，然后就爆发。

我們要建DREAM十萬人電報社區 https://t.me/dream_BSC
